content/en/docs/ops/diagnostic-tools/istioctl-analyze/test.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC2154
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e  # Exit on failure
+set -u  # Unset is an error
+# There is no need to echo, output appears in ops_diagnostic-tools_istioctl-analyze_test_debug.txt
+set -o pipefail
+
+# Use 'at' setup profile=default to install default profile?
+# This script doesn't need a control plane, but we need base CRDs
+# @setup profile=none
+
+# This command is allowed to fail
+snip_analyze_all_namespaces || true
+
+# The following step is 'usually' done by cleanup.  Do it here, just in case
+kubectl delete vs ratings || true
+
+snip_analyze_sample_destrule
+
+# TODO Check the output on failure for 'Referenced host+subset in destinationrule not found: "details+v2"' 
+_verify_failure snip_analyze_networking_directory
+
+snip_analyze_all_networking_yaml
+
+snip_analyze_all_networking_yaml_no_kube
+
+istioctl analyze --help
```

Comment:
Add check for some kind of expected output, using _verify_contains or _verify_lines, for example.